bot.js

```diff
@@ -51,12 +51,26 @@ client.on("disconnect", function() {
 })
 
 // Create an event listener for messages
-client.on('messageCreate', message => {
+client.on('messageCreate', async message => {
     if (client.user.id != message.author.id) {
-        ChatHandler.handle(message.content, message.author, message.channel, message);
+        await ChatHandler.handle(message.content, message.author, message.channel, message);
     }
 });
 
+process.on('uncaughtException', handleException);
```

Comment:
This is good, but you might want to remove the initial uncaughtException handling at the top of the bot file.

Reply:
Yeah, probably have missed that